README.md

```diff
@@ -1 +1,108 @@
-# Capstone_project
\ No newline at end of file
+
+<h3 id = "readme-top"><b>Capstone Project </b></h3>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+- [📖 Title](#Team Event)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Live Demo from loom](#Live-Loom-demo)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Credit](#Credit)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project] <a name="about-project">https://github.com/sam2if/Capstone_project.git</a>
+
+
+
+# Team Event
+This Capstone Project is based on an online website for a conference with a slogan "Reduce, Reuse and Recycle". 
+ You will learn how you can create enterprise value, empower greater worker productivity, and make the world a better place through advanced technology. 
+
+## Built With
+
+- Html
+- Css
+- JavaScript
+
+### Key Features <a name="key-features">
+- **[key_feature_1]** Capstone Project: built with CSS, Html and JavaScript.
+- **[key_feature_2]** Capstone Project: Mobile Version.
+- **[key_feature_3]** Capstone Project: The Home me page.
+- **[key_feature_4]** Capstone Project: desktop version.
+- **[key_feature_5]** Capstone Project: About me page for both mobile and desktop version.
+
+
+### Home page
+
+
+## Live Demo
+
+[Live Demo Link](https://sam2if.github.io/)
+
+## Live Loom demo
+
+[Live Demo from loom ](https://www.loom.com/share/71560190b49d49ab89b43cc9201f7a4f)
```

Comment:
- [ ] Hello @sam2if, great effort working on your capstone project, you have really done amazingly well. I would suggest we follow the project documentation and requirement as strict as possible as this will give enough confidence in our ability to build project. According to the project documentation, it was stated that you should show some piece of your code or part of the project you build that you are most proud of. but in the video presentation, you didn't show us any of the piece of your code or what you are most proud of that you have implemented in the project.

![Inkedloom](https://user-images.githubusercontent.com/78634841/225773827-55ae7080-b7f9-40dd-9b63-cce45bbabf69.jpg)